### DISCUSSION TITLE
Cannot read properties of undefined (reading 'index')

### DISCUSSION THREAD

**Discussion by PyTh0N22** [requarks/wiki#6741] [Error / Bug Report]
Hello.

I recently transferred the wiki.js backup to a new fresh server. I have the latest version and I was following the guide on docs.

I don't know why, but if I want to create, or edit a page, I always get this error: Cannot read properties of undefined (reading 'index').

The fun thing is that the content is saved. But it always displays this error.

**Comment by lepiou** [comment]
Hello, same error here

**Comment by lepiou** [comment]
Do you use Elasticsearch ? No problem with PostgreSQL

**Reply by PyTh0N22** [reply]
Yes. Elastisearch
